tensorboard/util/io_util.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""TensorBoard IO helpers."""
+
+
+def IsCloudPath(unused_path):
+    """Not implemented; simply returns False."""
```

Comment:
Let's preserve the open-source-relevant behavior of the existing helper, which has:

```
    return (
        path.startswith("gs://")
        or path.startswith("s3://")
        or path.startswith("/cns/")
    )
```

So we could drop `/cns/` but I would definitely keep the other two, since that benefits open source usage and we don't want to discard that benefit.

We can also preserve the corresponding tests (not that they are particularly complex tests :))

Reply:
Done. Thanks!